test/functional/p2p_invalid_messages.py

```diff
@@ -3,12 +3,14 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test node responses to invalid network messages."""
```

Comment:
> This sends a valid message.

Er... I'm very sorry if I'm being a pain in the ass, but why is this in p2p_invalid_messages?

Reply:
> I'm very sorry if I'm being a pain in the ass, but why is this in p2p_invalid_messages?

You're not. Because we couldn't think of anywhere better to put it 🤷‍♂️